src/aria/utils/Date.js

```diff
@@ -492,6 +492,7 @@ var ariaUtilsDate = module.exports = Aria.classDefinition({
             scope : this
         });
 
+        this._validateEntry = /^((?![a-z])\d(\s)?([\\h;m,.\-:\/\s])?(\s)?(\d)?(\s)?(am|pm|a\.m|p\.m|a\sm|p\sm|a.m.|p.m.|a. m|p. m|a .m|p .m|a . m|p . m)?(\s)?)+$/;
```

Comment:
Please turn capturing groups to non-capturing groups by adding `?:` right after the first parenthesis.
Also, removing unnecessary groups: `\s`, `\d` `[...]` are all atomic and the quantifiers behind them don't require a group in this case.
This will ease the reading of the expression.